core/templates/pages/admin-page/misc-tab/admin-misc-tab.component.ts

```diff
@@ -60,13 +76,138 @@ export class AdminMiscTabComponent {
   message: string = '';
   expIdToGetInteractionIdsFor!: string;
   explorationInteractionIds: string[] = [];
+  directiveSubscriptions = new Subscription();
+  loadingMessage: string = '';
+  selectedUserGroup: string = '';
+  newUserGroupName: string = '';
+  removable: boolean = true;
+  selectable: boolean = true;
```

Comment:
What is `removable` and `selectable`? Do both of these refer to user groups or the users in these groups? Maybe rename it to `userGroupIsRemovable` or `userIsRemovableFromUserGroup`. Ditto for `selectable`.

Reply:
Well come to think of it we would not be requiring these variables because as of now all the user-groups will be selectable and removable, hence I have removed them. Thanks for pointing it out!